packages/utils/src/get-awe-list.ts

```diff
@@ -0,0 +1,159 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { MetriportMedicalApi, PatientDTO } from "@metriport/api-sdk";
+import { executeAsynchronously } from "@metriport/core/util/concurrency";
+import { getEnvVar, getEnvVarOrFail } from "@metriport/core/util/env-var";
+import { out } from "@metriport/core/util/log";
+import { sleep } from "@metriport/core/util/sleep";
+import fs from "fs";
+import dayjs from "dayjs";
+import { Condition } from "@medplum/fhirtypes";
+
+const apiKey = getEnvVarOrFail("API_KEY");
+const apiUrl = getEnvVarOrFail("API_URL");
+const delayTime = parseInt(getEnvVar("BULK_INSERT_DELAY_TIME") ?? "200");
+const numberOfParallelExecutions = parseInt(getEnvVar("PARALLEL_QUERIES") ?? "10");
+const csvHeader = "id,firstName,lastName,dob,genderAtBirth,latestAWEDate,aweInPastYear\n";
+const curDateTime = new Date();
+const runName = `Z00s-${curDateTime.toISOString()}`;
+const baseDir = `./${runName}`;
+const patientWhitelist: string[] = [];
+
+function buildCSVFileName(): string {
+  return `${baseDir}/_Z00s.csv`;
+}
+
+const metriportAPI = new MetriportMedicalApi(apiKey, {
+  baseAddress: apiUrl,
+});
+
+// find latest Z00 codes per patient
+async function getAllAWEs() {
+  fs.mkdirSync(baseDir);
+  const csvFileName = buildCSVFileName();
+  fs.writeFileSync(csvFileName, csvHeader);
+  const errors: { patientId: string; error: Error }[] = [];
+
+  const facilities = await metriportAPI.listFacilities();
+  for (const facility of facilities) {
+    let patients = await metriportAPI.listPatients(facility.id);
+    patients = patients.filter(p => patientWhitelist.includes(p.id));
+    await executeAsynchronously(
+      patients,
+      async (patient, itemIndex, promiseIndex) => {
+        const { log } = out(`${promiseIndex + 1}-${itemIndex + 1}`);
+        try {
+          // list all of the patient's conditions
+          console.log(`>>> Getting Conditions for Patient ${patient.id}`);
+          const conditions = await metriportAPI.getPatientConsolidated(patient.id, ["Condition"]);
```

Comment:
This is quote expensive to run in parallel... maybe we set the default value to 5 instead of 10?